internal/ptr/utils.go

```diff
@@ -0,0 +1,13 @@
+package ptr
+
+func In[E any](e *E) E {
+	if e == nil {
+		return *new(E)
+	}
+
+	return *e
+}
+
+func Of[E any](e E) *E {
+	return &e
+}
```

Comment:
didn't we have this one already here? 🤔 
https://github.com/kubeflow/model-registry/blob/e712d445b076954d8f4c58acaf0eb69235702d40/internal/apiutils/api_utils.go#L50-L53

Reply:
Thanks I have missed this function :pray: